core/src/main/resources/lib/form/filter-menu-button/filter-menu-button.js

```diff
@@ -0,0 +1,56 @@
+function createFilterMenuButton (button, menu, menuAlignment, menuMinScrollHeight) {
```

Comment:
If these functions are to be global I'd suggest maybe namespacing them. I can see `createFilterMenuButton` being a name that could be easily overriden.

Reply:
it's already quite a long name, I think it will be okay for now anyway. Unless you've got any specific suggestions?